src/pages/Members/styles.scss

```diff
@@ -80,3 +42,41 @@
     }
   }
 }
+
+.table {
```

Comment:
As this is now specific to the Table component it should be moved to the components/Table. Or if this is override for the Members table, then it should stay here